packages/ckeditor5-list/src/listproperties/listpropertiesui.ts

```diff
@@ -213,6 +228,7 @@ function getDropdownViewCreator( {
 	buttonIcon: string;
 	styleGridAriaLabel: string;
 	styleDefinitions: Array<StyleDefinition>;
+	isStyleVisible: boolean;
```

Comment:
This won't be needed if you pass a normalized config in the `propertiesConfig` parameter.